src/index.html

```diff
@@ -0,0 +1,127 @@
+          </div>
+        </div>
+
+
+        <div class="content" id="profile-bio">
+          <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin ornare magna eros, eu pellentesque tortor
+            vestibulum ut.</p>
+        </div>
+      </div>
+    </div>
+
+    <div class="events-container column">
+      <h2 class="subtitle is-4">History</h2>
+
+      <div class="timeline" id="user-timeline">
+
+        <div class="timeline-item">
+          <div class="timeline-marker"></div>
+          <div class="timeline-content">
+            <p class="heading">Jan 21, 2016</p>
+            <div class="content">
+                  <span class="gh-username">
+                    <img src="https://avatars.githubusercontent.com/u/2310778?"/>
+                    <a href="https://github.com/sethii">sethii</a>
+                  </span>
+              closed
+              <a href="https://github.com/TheSoftwareHouse/Kakunin/pull/41">pull request</a>
+              <p class="repo-name">
+                <a href="https://github.com/TheSoftwareHouse/Kakunin">TheSoftwareHouse/Kakunin</a>
+              </p>
+            </div>
+          </div>
+        </div>
+
+        <div class="timeline-item is-primary">
+          <div class="timeline-marker is-primary"></div>
+          <div class="timeline-content">
+            <p class="heading">Jan 19, 2016</p>
+            <div class="content">
+                <span class="gh-username">
+                  <img src="https://avatars.githubusercontent.com/u/1208786?"/>
+                  <a href="https://github.com/simonsipl">gajdus</a>
+                </span>
+              created
+              <a href="https://github.com/TheSoftwareHouse/Kakunin/pull/41#discussion_r176273897">comment</a>
+              to
+              <a href="https://api.github.com/repos/TheSoftwareHouse/Kakunin/pulls/comments/176273897">pull request</a>
+              <p class="repo-name">
+                <a href="https://github.com/TheSoftwareHouse/Kakunin">TheSoftwareHouse/Kakunin</a>
+              </p>
+            </div>
+          </div>
+        </div>
+
+        <div class="timeline-item">
+          <div class="timeline-marker"></div>
+          <div class="timeline-content">
+            <p class="heading">Jan 19, 2016</p>
+            <div class="content">
+                <span class="gh-username">
+                  <img src="https://avatars.githubusercontent.com/u/2310778"/>
+                  <a href="https://github.com/aherok">sethii</a>
+                </span>
+              opened
+              <a href="https://github.com/TheSoftwareHouse/Kakunin/pull/41">pull request</a>
+              <p class="repo-name">
+                <a href="https://github.com/TheSoftwareHouse/Kakunin">TheSoftwareHouse/Kakunin</a>
+              </p>
+            </div>
+          </div>
+        </div>
+
+
+      </div>
+
+    </div>
+  </div>
+</div>
+
+</body>
+</html>
```

Comment:
It is nice to have good intendation between HTML tags, when nesting like in XML nesting, it is easier to read which is nested where. I know that body && head && html tags are obvious, but.. nice to have it nested properly